### PR TITLE
Fix integrity when migrating from 9.1.0-9.1.3

### PR DESCRIPTION
**Pull request by VicDeo**
Fixing issue reported at
https://github.com/owncloud/administration/pull/111#issuecomment-283940241

extra file `updater/.gitignore` is left after update from any 9.1.x to 9.1.4

### CODE REVIEW

File: src/Tests/bootstrap.php [hunk 1]
```diff
@@ -6,5 +6,6 @@
 require_once __DIR__.'/../../vendor/autoload.php';
 
 if(!class_exists('PHPUnit_Framework_TestCase')) {
-	require_once('PHPUnit/Autoload.php');
+	class PHPUnit_Framework_TestCase extends \PHPUnit\Framework\TestCase {
```
Comment (DeepDiver1975) [inline src/Tests/bootstrap.php]: no - please setup phpunit dependency properly in composer.json

**Comment by VicDeo** [comment]
@DeepDiver1975 I don't want to pollute vendor/* because phpunit depends on a bunch of other packages and brings a lot of PITA for whatever reasons.
I've added it to  `.travis` instead

**Comment by davitol** [comment]
:+1: WFM

**Comment by VicDeo** [comment]
@davitol I think it should be reproducible while updating from 9.0.x using the following path:
(stable channel) 9.0.4 -> 9.0.8  
(daily channel) -> 9.1.4

Backport  to 9.0.x follows

**Comment by VicDeo** [comment]
Stable9: https://github.com/owncloud/updater/issues/418

**Pull request merged**